components/datatypes/poi-detail.schema.json

```diff
@@ -144,6 +144,11 @@
           },
           "meta:titleId": "poi-detail##xdm:geoInteractionDetails##title##47521",
           "meta:descriptionId": "poi-detail##xdm:geoInteractionDetails##description##39951"
+        },
+        "xdm:metadata": {
+          "title": "POI metadata",
+          "description": "metadata assigned to the POI by the administrator of the POI definitions.",
+          "$ref": "https://ns.adobe.com/experience/analytics/keyedlist"
```

Comment:
@iewilliam , I noticed here that you are using a data type from Analytics here which is not recommended as we do not manage the functional changes of schema for internal Adobe solutions. It's very common to have standard XDM components used in the solution Field groups but not the other way around. Please clone the analytics data type under components/datatypes folder and then use that in your $ref.

Reply:
done @mjasrotia